### PR TITLE
More tests added

### PR DESCRIPTION
**Pull request by mgs95**

### CODE REVIEW

File: app/ui/src/Components/ProjectsBar/components/ProjectsOrder/ProjectsOrder.test.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    apolloRender(Component, mockClient);
+  });
+
+  it('should render without crashing', () => {
+    expect(screen.getByText('Creation date')).toBeInTheDocument();
+    expect(getSnapshot()).toMatchSnapshot();
+  });
+
+  it('should allow the selection of other options', function () {
+    // Arrange.
```
Comment (marco-gattelli) [inline app/ui/src/Components/ProjectsBar/components/ProjectsOrder/ProjectsOrder.test.tsx:49]:
```suggestion
```
we can remove it if we want :)

File: app/ui/src/Mocks/entities/user.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { apiToken1 } from './apiToken';
+import { AccessLevel } from '../../Graphql/types/globalTypes';
```
Comment (marco-gattelli) [inline app/ui/src/Mocks/entities/user.ts:2]:
```suggestion
import { AccessLevel } from 'Graphql/types/globalTypes';
```

File: app/ui/src/Components/SettingsMenu/SettingsMenu.test.tsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+let mockClient;
+beforeEach(() => {
+  mockClient = createMockClient();
+});
+afterEach(cleanup);
+
+describe('When component is loading', () => {
+  it('should render without crashing', () => {
+    // Arrange.
```
Comment (marco-gattelli) [inline app/ui/src/Components/SettingsMenu/SettingsMenu.test.tsx:35]:
```suggestion
```
same

File: app/ui/src/Components/SettingsMenu/SettingsMenu.test.tsx [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+  });
+
+  describe('When opening settings', () => {
+    // Opens settings
+    beforeEach(() => {
+      fireEvent.click(screen.getByText(data.me.email, { exact: false }));
+    });
+
+    it('should not perform any action when clicking on the separator', () => {
+      // Arrange.
```
Comment (marco-gattelli) [inline app/ui/src/Components/SettingsMenu/SettingsMenu.test.tsx:65]:
```suggestion
```
same

File: app/ui/src/Components/SettingsMenu/SettingsMenu.test.tsx [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+      fireEvent.click(screen.getByText('USER SETTINGS', { exact: false }));
+
+      // Assert.
+      expect(
+        screen.getByText('USER SETTINGS', { exact: false })
+      ).toBeInTheDocument();
+    });
+
+    it('should redirect to ssh key page when clicking on this section', () => {
+      // Arrange.
```
Comment (marco-gattelli) [inline app/ui/src/Components/SettingsMenu/SettingsMenu.test.tsx:76]:
```suggestion
```
same

File: app/ui/src/Components/SiteBar/SiteBar.test.tsx [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+  test('Component match snapshot', () => {
+    expect(component).toMatchSnapshot();
+  });
+
+  it('should hide navigation buttons when running on browser', function () {
+    expect(component.find(ArrowsNavigator).isEmptyRender()).toBeTruthy();
+  });
+
+  it('should show navigation buttons when running on electron', function () {
+    // Arrange.
```
Comment (marco-gattelli) [inline app/ui/src/Components/SiteBar/SiteBar.test.tsx:21]:
```suggestion
```
same

File: app/ui/src/Pages/Project/components/ProjectNavigation/components/NavElements/NavElements.tsx [hunk 7]
```diff
@@ -85,7 +88,7 @@ function NavElements({ isOpened }: Props) {
               warning
             >
               <NavigationButton
-                label={areToolsActive ? 'Stop tools' : 'Run tools'}
+                label="Stop tools"
```
Comment (marco-gattelli) [inline app/ui/src/Pages/Project/components/ProjectNavigation/components/NavElements/NavElements.tsx:91]: why do you remove the `Run tools` label? Is it normal to show the `play` icon and the label is `Stop tools`
Captura de pantalla 2021-05-10 a las 15 10 58

File: app/ui/src/Pages/Projects/Projects.test.tsx [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+  });
+
+  it('should render without crashing', () => {
+    expect(screen.getByText(ERROR_MESSAGE)).toBeInTheDocument();
+  });
+});
+
+// Arrange.
+// Act.
+// Assert.
```
Comment (marco-gattelli) [inline app/ui/src/Pages/Projects/Projects.test.tsx:125]:
```suggestion
```
can we remove these three lines?

**Review by marco-gattelli — APPROVED** [review]

File: app/ui/src/Components/ProjectsBar/components/ProjectsOrder/ProjectsOrder.test.tsx (hunk 1 above)
Comment (mgs95) [reply app/ui/src/Components/ProjectsBar/components/ProjectsOrder/ProjectsOrder.test.tsx:49]: Done

File: app/ui/src/Mocks/entities/user.ts (hunk 2 above)
Comment (mgs95) [reply app/ui/src/Mocks/entities/user.ts:2]: Done

File: app/ui/src/Pages/Project/components/ProjectNavigation/components/NavElements/NavElements.tsx (hunk 7 above)
Comment (mgs95) [reply app/ui/src/Pages/Project/components/ProjectNavigation/components/NavElements/NavElements.tsx:91]: Ups, that was an small slip. Fixed.

**Pull request merged**